packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -274,6 +274,49 @@ export interface RaiseHandCallFeatureState {
   localParticipantRaisedHand?: RaisedHandState;
 }
 
+/* @conditional-compile-remove(together-mode) */
+/**
+ * State only version of {@link @azure/communication-calling#TogetherModeCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for raised hands on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ * @alpha
+ */
+export interface TogetherModeCallFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#TogetherModeCallFeature.togetherModeStream}.
+   */
+  stream?: TogetherModeStreamState;
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * State only version of {@link @azure/communication-calling#TogetherModeVideoStream}.
+ * @alpha
+ */
+export interface TogetherModeStreamState {
+  /**
+   * Proxy of {@link @azure/communication-calling#TogetherModeVideoStream.id}.
+   */
+  id: number;
```

Comment:
note: most streamIds are strings currently. Keep this the same type as the calling sdk type but just making a note here in case you wanted to alter the calling sdk type to be consistent